gha.dist.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       fail-fast: false
       matrix:
         php: ['7.4', '8.0', '8.1']
```

Comment:
Why have you not changed this in line with https://moodledev.io/general/releases/4.5 ?